src/libs/elektra/keyset.c

```diff
@@ -846,15 +847,21 @@ if (result >= 0)
  *
  * @param ks the keyset to work with
  * @param key the key to check
+ * @param options lookup options to use
  * @return position where the key is (>=0) if the key was found
  * @return -insertpos -1 (< 0) if the key was not found
  *    so to get the insertpos simple do: -insertpos -1
  * @see ksLookup() for retrieving the found key
  */
-ssize_t ksSearch (const KeySet * ks, const Key * key)
+ssize_t ksSearch (const KeySet * ks, const Key * key, elektraLookupOptions options)
```

Comment:
The correct type is probably `elektraLookupFlags`.

Reply:
Don't we need `elektraLookupOptions` from `kdbhelper.h` to check whether we should use OPMPHM?

Reply:
No, the flags are only to enforce building up the hash. By default there is a predictor which has heuristics for rebuilding.